Custom Device Source/Scripting Examples/System Definition Files/Base.nivssdf

```diff
@@ -0,0 +1,1964 @@
+							<Properties />
+							<Errors />
+						</Section>
+						<Section Name="DAQ" TypeGUID="77550612-1485-13A6-56570AB5032158E9">
+							<Description />
+							<Properties />
+							<Errors />
+							<Section Name="Waveform Tasks" TypeGUID="1D6B8779-A957-42D1-B074-82153DB7FF44">
+								<Description />
+								<Properties />
+								<Errors />
+							</Section>
+						</Section>
+						<Section Name="FPGA" TypeGUID="03D3B659-1485-13A6-5652D8A07328ECB7">
+							<Description />
+							<Properties />
+							<Errors />
+						</Section>
+						<Section Name="NI-XNET" TypeGUID="6b693e6d-67b1-40b8-8108-4a4cc3147721">
+							<Description />
+							<Properties>
+								<Property Name="CD Status">
+									<U32>2</U32>
+								</Property>
+								<Property Name="Decimation">
+									<I32>1</I32>
+								</Property>
+								<Property Name="Driver VI Exec Mode">
+									<U32>1</U32>
+								</Property>
+							</Properties>
+							<Errors />
+							<Section Name="CAN" TypeGUID="6c5bb91f-0da7-407d-aefc-acbe49c768df">
+								<Description />
+								<Properties />
+								<Errors />
+							</Section>
+							<Section Name="FlexRay" TypeGUID="e33fae90-03f1-4f76-b2f1-6eb8fcac2676">
+								<Description />
+								<Properties />
+								<Errors />
+							</Section>
+							<Section Name="LIN" TypeGUID="c9c30728-42cb-4dc3-b054-b80e7a54e833">
+								<Description />
+								<Properties />
+								<Errors />
+							</Section>
+						</Section>
+						<Section Name="Data Sharing" TypeGUID="d56f3ac1-7b2c-422c-9d11-fd9cc7abf2a5">
+							<Description />
+							<Properties>
+								<Property Name="CD Status">
+									<U32>2</U32>
+								</Property>
+							</Properties>
+							<Errors />
+						</Section>
+					</Section>
+					<Section Name="SLSC" TypeGUID="3ea8ee87-daf4-4abc-a6e6-9c54a9452824">
+						<Description />
+						<Properties />
+						<Errors />
+						<Section Name="SLSC Chassis" TypeGUID="2d230194-1f79-4241-ad00-e5b0a0d634cb">
+							<Description />
+							<Properties>
+								<Property Name="Chassis Type">
+									<String>SLSC-12001</String>
+								</Property>
+								<Property Name="Chassis ID">
+									<String>slsc-12001-00000000</String>
+								</Property>
+								<Property Name="Username">
+									<String>anonymous</String>
+								</Property>
+								<Property Name="Password">
+									<String />
+								</Property>
+								<Property Name="Chassis ID Type">
+									<I32>1</I32>
+								</Property>
+							</Properties>
+							<Errors />
+							<Section Name="Modules" TypeGUID="96319964-d29d-4af0-9c60-b1a785679b5f">
+								<Description />
+								<Properties />
+								<Errors />
+								<Section Name="Slot 1 (Empty)" TypeGUID="8a754f70-4d3d-42e2-9846-157cb4981bb4">
+									<Description />
+									<Properties />
+									<Errors />
+								</Section>
+								<Section Name="Slot 2 (SLSC Switch)" TypeGUID="5e47c8de-3ec6-490b-8897-5526d4aea756">
```

Comment:
This system definition should be re-created from scratch, to remove non-Scan Engine items (like SLSC Switch).

Reply:
Humm... good point.